### ISSUE TITLE
[FEATURE REQUEST] Skip silence on playback

### ISSUE THREAD

**Issue by grappas** [futo-org/grayjay-android#864]
This option in NewPipe is awesome. Looking forward to see it also in Grayjay :)
https://github.com/TeamNewPipe/NewPipe/discussions/10756

**Comment by Mobious** [comment]
+1
The current lack of this feature is the main reason I haven't fully switched to Grayjay from NewPipe.

**Comment by PacoBell** [comment]
Both [Just (Video) Player](https://github.com/moneytoo/Player) and now [Next Player](https://github.com/anilbeesetti/nextplayer) support ExoPlayer's skipSilenceEnabled. It shouldn't be too hard to enable this in Grayjay, either.

**Comment by grappas** [comment]
Bump

**Comment by galeofperil** [comment]
bump

**Comment by Mxfuuu** [comment]
I'd love to see this feature on Grayjay as well. I look forward to it.
Unfortunately I am having to install separate apps for this feature.

**Comment by Laurent2916** [comment]
+1
This feature is the only thing keeping me from switching as well!
Looking at [newpipe's code](https://github.com/TeamNewPipe/NewPipe/blob/9ba89d418b444b127ee936f7a39dc5c5b54521ec/app/src/main/java/org/schabi/newpipe/player/Player.java#L977-L994) and [just player's code](https://github.com/moneytoo/Player/blob/38f1f0ee384c8726759af7ed807d0cb50f65d4c1/app/src/main/java/com/brouken/player/PlayerActivity.java#L1255-L1257), [like said above](https://github.com/futo-org/grayjay-android/issues/864#issuecomment-2393188120), it seems like it is only a matter of using [`setSkipSilenceEnabled`](https://developer.android.com/reference/kotlin/androidx/media3/exoplayer/ExoPlayer#setSkipSilenceEnabled(boolean)).

**Comment by defiling9046** [comment]
Using the SoundCloud plugin on Grayjay to play music and this would be very handy, for example, at the end of songs when they sometimes have a few seconds of silence. Thank you Grayjay team for making this great app in the first place.🎉

**Comment by oddtoddy** [comment]
Yeah, NewPipe does it extremely well. Makes getting through podcasts and spoken word videos much quicker.

**Comment by defiling9046** [comment]
If this could be customised by decibel as well that would be great as well as some songs at the end or start have it fade out with like 1 decibel of the song still playing so to skip that would be nice.👍

**Comment by supanchito** [comment]
I was about to make the switch to Grayjay because I listen to a lot of podcasts and discussions. However, since saving time is important to me and I noticed that this feature isn't implemented, I'm held back. Please, look into it!
